thunder/dynamo/utils.py

```diff
@@ -16,6 +16,7 @@
 from thunder.torch import _torch_to_thunder_function_map
 from thunder.torch.langctx import torchctx
 from thunder.core.utils import check
+import thunder
```

Comment:
Is it a leftover from some experimentation? This import is unused in the file.

Reply:
yes, will fix